.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.1
```

Comment:
Usually, you should NOT change ruby version of a project without clearance from other project members or project lead (in this case your coaches) . Just install whatever ruby version the project is asking for.